### ISSUE TITLE
[Bug] The data nodes should be displayed on the "source" and "sink" sides

### ISSUE THREAD

**Issue by LiJie20190102** [apache/inlong#9252]
### What happened

According to the official website, "Data Nodes" should include "Extract Nodes" and "Load Nodes". However, when creating the "Ingestion" task, data nodes cannot be selected on the "source" side, but can be displayed on the "sink" side.

source:
![image](https://github.com/apache/inlong/assets/53458004/34217021-a4ab-4c82-802a-486971eae20a)

sink:
![image](https://github.com/apache/inlong/assets/53458004/9af69792-164a-43fb-b21a-b438e38961e3)

### What you expected to happen

The ‘’data nodes‘’ should be displayed on the "source" and "sink" sides

### How to reproduce

Create the 'Ingestion' task and check if there are any ‘data nodes‘’ in the 'source'

### InLong version

1.9.0

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by dockerzhang** [comment]
@LiJie20190102 You are right, all of the `source` should use data nodes too, we plan to finish them in the future, thanks.

**Comment by aloyszhang** [comment]
This issue has been open for a long time with no activity, close it first and feel free to re-open it if needed.
